### DISCUSSION TITLE
[OSPP-Week4] Debug and write PLAS algorithm

### DISCUSSION THREAD

**Discussion by pilgrimygy** [JuliaReinforcementLearning/ReinforcementLearning.jl#409] [Archive]
@findmyway 
[Last week plan](https://github.com/JuliaReinforcementLearning/ReinforcementLearning.jl/discussions/388)

Last week, I did the work:

I will try to complete the continuous version of the Critic Regularized Regression (CRR) algorithm. But its performance is poor.

Week 4:
1. I will fix the bugs in Continuous CRR.
2. Begin to write the PLAS algorithm.
3. Consider adding a new component (for example, VAE) into RLCore.

**Comment by findmyway** [comment]
Cool, let me know if you need any help in debugging the performance issue.

**Reply by pilgrimygy** [reply]
https://github.com/JuliaReinforcementLearning/ReinforcementLearning.jl/pull/407

Maybe the `actor_loss_coef` calculation is wrong. If it doesn't use `actor_loss_coef` to weight, the algorithm will be similar to Behavior Cloning and its performance will be slightly better.
